### PR TITLE
Use an official version of Prometheus instead of master.

### PR DESCRIPTION
**Pull request by songy23**
Also run go mod tidy.

/cc @odeke-em

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,6 +1,7 @@
 module contrib.go.opencensus.io/exporter/prometheus
 
 require (
-	github.com/prometheus/client_golang v0.9.3-0.20190127221311-3c4408c8b829 // TODO: change the version to v0.9.2 once oc-go version is updated.
+	github.com/prometheus/client_golang v0.9.2
 	go.opencensus.io v0.21.0
+
```
Comment (odeke-em) [inline go.mod]: Spurious newline. Was this manually added? If so, in the future, you can do
```go
GO111MODULE=on go get -u -v github.com/prometheus/client_golang@v0.9.2
```
and then `go mod tidy`

File: go.mod [hunk 2]
```diff
@@ -1,6 +1,7 @@
 module contrib.go.opencensus.io/exporter/prometheus
 
 require (
-	github.com/prometheus/client_golang v0.9.3-0.20190127221311-3c4408c8b829 // TODO: change the version to v0.9.2 once oc-go version is updated.
```
Comment (odeke-em) [inline go.mod:4]: What was the reason for this pin to that specific version?

Comment (songy23) [reply go.mod:4]: https://github.com/census-instrumentation/opencensus-service/pull/512#discussion_r271958614.

File: go.mod (hunk 1 above)
Comment (songy23) [reply go.mod]: Removed, seems to be added by accident.

**Review by rghetia — APPROVED** [review]

**Pull request merged**